cmd/adduser.go

```diff
@@ -311,3 +266,98 @@ func (p *AddUserParams) editUserClaim(c interface{}, ctx ActionCtx) error {
 
 	return nil
 }
+
+type ResponsePermsParams struct {
+	respTTL string
+	respMax string
+	rmResp  bool
+}
+
+func (p *ResponsePermsParams) maxResponseValidator(s string) error {
```

Comment:
Just checked with Derek and values `<0` should be rejected (0 is ok, server will use default).

Reply:
Retracting that statement, will use negative in the server to make the value "infinite"